src/data/screenshots.json

```diff
@@ -219,201 +219,201 @@
-                    { "filename": "/assets/screenshots/2-10/en/ios/rat_6", "alt": "Rapid antigen test image 6 of 10" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/rat_7", "alt": "Rapid antigen test image 7 of 10" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/rat_8", "alt": "Rapid antigen test image 8 of 10" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/rat_9", "alt": "Rapid antigen test image 9 of 10" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/rat_10", "alt": "Rapid antigen test image 10 of 10" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_1", "alt": "Rapid antigen test image 1 of 10" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_2", "alt": "Rapid antigen test image 2 of 10" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_3", "alt": "Rapid antigen test image 3 of 10" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_4", "alt": "Rapid antigen test image 4 of 10" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_5", "alt": "Rapid antigen test image 5 of 10" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_6", "alt": "Rapid antigen test image 6 of 10" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_7", "alt": "Rapid antigen test image 7 of 10" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_8", "alt": "Rapid antigen test image 8 of 10" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_9", "alt": "Rapid antigen test image 9 of 10" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/rat_10", "alt": "Rapid antigen test image 10 of 10" }
                 ]
             },
             {
                 "title": "Certificates",
                 "anchor": "ios_certificates",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_health_certificate_empty_screen", "alt": "Certificates image 1 of 4" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_health_certificate_consent_screen", "alt": "Certificates image 2 of 4" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_family_certificate-cert-1", "alt": "Certificates image 3 of 4" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_family_certificate-cert-2", "alt": "Certificates image 4 of 4" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_health_certificate_empty_screen", "alt": "Certificates image 1 of 4" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_health_certificate_consent_screen", "alt": "Certificates image 2 of 4" },
```

Comment:
Image one and two should be flipped, the app first shows the consent screen and after clicking on "Continue" the app shows the "Empty certificates tab" screen.

Reply:
@Ein-Tim Thanks!